tasks/eva_3004/eva3004/pom.xml

```diff
@@ -58,9 +58,6 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
                 <version>3.1.0</version>
-                <configuration>
-                    <executable>java</executable>
-                </configuration>
```

Comment:
This executable can be supplied at runtime and can be made flexible (java or groovy). Hence we don't have to hardcode it in the POM.